test-data/unit/typexport-basic.test

```diff
@@ -997,6 +997,18 @@ A.f(ab, o)
 [out]
 MemberExpr(10) : def (self: A[B], y: builtins.object)
 
+[case testClassMethodOfGenericClass]
```

Comment:
Why are you using a `typexport` test instead of "normal" `check-*.test` tests?

Reply:
I have not consciously thought. Sorry. I added it to `typexport-basic.test` because it has some generics tests already.

Is it better if I create new file like `test-data/unit/check-classmethods.test` ?
Or is there some better place?

Reply:
I changed my mind. I will put it to `check-generics.test`. It's more narrow than classes.